systems.csv

```diff
@@ -206,6 +206,11 @@ DE,Roxy Freiburg,"Freiburg im Breisgau, DE",1227,https://roxy-mobility.com,https
 DE,RSVG-Bike,"RSVG, DE",nextbike_rb,https://www.nextbike.de/de/rsvg/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_rb/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_rb%2Fgbfs.json,
 DE,RVK,"RVK e-Bike, DE",nextbike_dr,https://www.nextbike.de/de/rvk/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_dr/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_dr%2Fgbfs.json,
 DE,SAP Walldorf,"Walldorf, DE",nextbike_ds,https://www.nextbike.de/de/walldorf/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_ds/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_ds%2Fgbfs.json,
+DE,Shared Mobility GBFS – CallABike,DE,cab_germany,https://www.callabike.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABike/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FCallABike%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start 
+DE,Shared Mobility GBFS – CallABikeICE,DE,cab_ice_stationen,https://www.callabike.de/de/ice-stationen,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABikeICE/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FCallABikeICE%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
+DE,Shared Mobility GBFS – StadtRadHamburg,"Hamburg, DE",stadtrad_hamburg,https://stadtrad.hamburg.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/StadtRadHamburg/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FStadtRadHamburg%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
+DE,Shared Mobility GBFS – RegioRadStuttgart,"Stuttgart, DE",regiorad_stuttgart,https://www.regioradstuttgart.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/RegioRadStuttgart/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FRegioRadStuttgart%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
+DE,Shared Mobility GBFS – StadtRADLueneburg,"Lübeburg, DE",stadtrad_lueneburg,https://stadtradlueneburg.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/StadtRADLueneburg/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FStadtRADLueneburg%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
```

Comment:
I would suggest to remove `Shared Mobility GBFS – ` from the system name as all records in this catalog are shared mobility GBFS feeds. Thank you!

```suggestion
DE,CallABike,DE,cab_germany,https://www.callabike.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABike/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FCallABike%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start 
DE,CallABikeICE,DE,cab_ice_stationen,https://www.callabike.de/de/ice-stationen,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABikeICE/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FCallABikeICE%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
DE,StadtRadHamburg,"Hamburg, DE",stadtrad_hamburg,https://stadtrad.hamburg.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/StadtRadHamburg/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FStadtRadHamburg%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
DE,RegioRadStuttgart,"Stuttgart, DE",regiorad_stuttgart,https://www.regioradstuttgart.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/RegioRadStuttgart/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FRegioRadStuttgart%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
DE,StadtRADLueneburg,"Lübeburg, DE",stadtrad_lueneburg,https://stadtradlueneburg.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/StadtRADLueneburg/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FStadtRADLueneburg%2Fgbfs,https://developers.deutschebahn.com/db-api-marketplace/apis/start
```